pygubudesigner/bindingseditor.py

```diff
@@ -15,10 +15,11 @@
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import unicode_literals
+
 try:
     import tkinter as tk
     import tkinter.ttk as ttk
-except:
+except BaseException:
```

Comment:
```suggestion
except Exception:
```

Should be a plain `Exception` instead of `BaseException`